src/vm/runners/cairo_runner.rs

```diff
@@ -376,10 +364,8 @@ impl CairoRunner {
             .map_err(VirtualMachineError::TracerError)?;
         vm.end_run(&self.exec_scopes)?;
 
-        if !disable_finalize_all {
-            vm.segments.compute_effective_sizes(&vm.memory);
-            self.run_ended = true;
-        }
+        vm.segments.compute_effective_sizes(&vm.memory);
+        self.run_ended = true;
```

Comment:
This should still be supported. What I meant we don't need to try is to use `Option<_>` to give a default, as the point is not having to pass any argument. Just pass the `bool`.